src/pages/workspaces/WorkspacesList/RenderedWorkspaces.ts

```diff
@@ -386,7 +386,8 @@ const ActionsCell = (props: ActionsCellProps): ReactNode => {
     workspaceId: string,
     policies?: WorkspacePolicy[],
     bucketName?: string,
-    description?: string
+    description?: string,
+    googleProject?: string
```

Comment:
Unfortunately I still can't think of a reasonable way to unittest the changes in this file (to ensure that we are properly extending the workspace object with the passed values).

Reply:
For most of the components in here, it feels almost silly to test them individually, because they're just displaying some workspace attribute with minimal transformation, and hardly any behavior to test. So the baseline level of reward for testing feels lower.

`ActionsCell` on the other hand, is a pain to mock, so it makes the overall cost higher.

Combine that with the fact that most of these components aren't exported individually, and the behavior contained of `ActionsCell` isn't very exposed, and it feels very daunting and pointless.

Exporting individual components or splitting them up into their own files (which has the same effect), in order to test them in isolation might improve matters.